CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.0.1 - November 10, 2021
+This is a patch release on top of `v2.0.0` which does not include changes introduced in `v2.1.0-beta.1` and later. The `v2.1.0-beta.2` remains the latest and most up-to-date release.
+
+For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+### Changelog
+#### Bug fixes and improvements
+- Refactored code to update traffic data based on congestion numeric for route refresh API calls. [#5091](https://github.com/mapbox/mapbox-navigation-android/pull/5091/files)
+- Implemented predictive cache with `TilesetDescriptor` so that volatile sources are not loaded unexpectedly. Warning: `PredictiveCacheController.createStyleMapControllers` method needs to be used in order to cache only non-volatile styles. [#5089](https://github.com/mapbox/mapbox-navigation-android/pull/5089)
```

Comment:
```suggestion
- Implemented predictive cache with `TilesetDescriptor` so that volatile sources are not loaded unexpectedly. Warning: `PredictiveCacheController.createStyleMapControllers` method needs to be used in order to cache only non-volatile sources. [#5089](https://github.com/mapbox/mapbox-navigation-android/pull/5089)
```